### PR TITLE
Fix CircleCI Percy errors

### LINKED ISSUE #137: Fix percy build
Need to use `--ignore-files` in CircleCI build.

https://docs.percy.io/docs/jekyll#section-ignoring-files

### PR DESCRIPTION
**Pull request by spectralapps**
fixes #137

### CODE REVIEW

**Comment by spectralapps** [comment]
check build details
percy: success 
we should avoid using --widths [1280]
The default value already is Default: [375, 1280]

File: .circleci/config.yml [hunk 1]
```diff
@@ -127,7 +127,7 @@ jobs:
           name: Run percy snapshots
           command: |
             bundle exec jekyll build
-            npx percy snapshot _site/ --widths "1280" --ignore-files "assets/python/**"
+            npx percy snapshot _site/ --ignore-files "assets/python/**"
```
Comment (jbampton) [inline .circleci/config.yml]: Also need to ignore -> `parallel-universe/index.html` as this page is very long and this times out the percy build.

File: .circleci/config.yml [hunk 2]
```diff
@@ -127,7 +127,7 @@ jobs:
           name: Run percy snapshots
           command: |
             bundle exec jekyll build
-            npx percy snapshot _site/ --widths "1280" --ignore-files "assets/python/**"
+            npx percy snapshot _site/ --ignore-files "assets/python/**, parallel.html"
```
Comment (jbampton) [inline .circleci/config.yml]: I think the file is called `parallel-universe/index.html` not `parallel.html`

**Comment by jbampton** [comment]
Screen Shot 2019-12-10 at 8 57 57 am

**Pull request merged**